Gruntfile.js

```diff
@@ -89,6 +89,8 @@ module.exports = function( grunt ) {
           "font-sizes": false,
           "ids": false,
           "important": false,
+          "known-properties": false,
```

Comment:
let's remove this and fix the less errors (it seems like they are mostly all `filter: alpha(opacity);` which we can safely change to `opacity: ##`). Note that when you change filter to opacity, you will need to divide the filter value by 100, so for e.g. `45` will become `0.45`.

There is one error generated by normalize.less. We should probably not lint that file since they lint it already (we probably want to switch to npm installing this at some point). You just need to add an entry to `src` above with `"!./public/editor/stylesheets/normalize.less"`